lib/usd/translators/meshWriter.cpp

```diff
@@ -41,19 +43,107 @@
 #include <pxr/usd/usdSkel/root.h>
 #include <pxr/usd/usdUtils/pipeline.h>
 
+#include <maya/MFnBlendShapeDeformer.h>
 #include <maya/MFnDependencyNode.h>
+#include <maya/MFnGeometryFilter.h>
 #include <maya/MFnMesh.h>
+#include <maya/MGlobal.h>
 #include <maya/MIntArray.h>
+#include <maya/MItDependencyGraph.h>
 #include <maya/MPlug.h>
 #include <maya/MPlugArray.h>
 #include <maya/MString.h>
 #include <maya/MStringArray.h>
 #include <maya/MUintArray.h>
 
+#include <cassert>
+#include <cfloat>
 #include <set>
 #include <string>
 #include <vector>
 
+MObject mayaFindOrigMeshFromBlendShapeTarget(const MObject& mesh, MObjectArray* intermediates)
+{
+    assert(mesh.hasFn(MFn::kMesh));
```

Comment:
Remove assert.

Reply:
This, and other asserts, have been addressed in 6460336.